wodles/aws/aws_s3.py

```diff
@@ -2592,6 +2611,10 @@ def iter_files_in_bucket(self, aws_account_id: str = None, aws_region: str = Non
                         debug(f"+++ Remove file from S3 Bucket:{bucket_file['Key']}", 2)
                         self.client.delete_object(Bucket=self.bucket, Key=bucket_file['Key'])
                     self.mark_complete(aws_account_id, aws_region, bucket_file)
+                    processed_logs += 1
+
+                if processed_logs == 0:
+                    self._print_not_logs_to_prcess_message(self.bucket, aws_account_id, aws_region)
```

Comment:
Typo in the method's call
```suggestion
                    self._print_not_logs_to_process_message(self.bucket, aws_account_id, aws_region)
```